src/xml/ConfigParser.cpp

```diff
@@ -129,9 +129,7 @@ int ConfigParser::readXmlFile(std::string const &filePath)
   SAXHandler.serror         = OnStructuredErrorFunc;
 
   std::ifstream ifs(filePath);
-  if (not ifs) {
-    PRECICE_ERROR("File open error: " << filePath);
-  }
+  PRECICE_CHECK(ifs, "XML parser was unable to open configuration file \"" << filePath << '"');
```

Comment:
Why does this usually happen? Can we help the user?

Reply:
https://github.com/precice/precice/pull/740#discussion_r454384516